packages/usa-date-picker/src/test/date-picker-i18n.spec.js

```diff
@@ -0,0 +1,183 @@
+const fs = require("fs");
+const path = require("path");
+const assert = require("assert");
+const EVENTS = require("./events");
+const DatePicker = require("../index");
+
+const TEMPLATE = fs.readFileSync(
+  path.join(__dirname, "/date-picker-i18n.template.html")
+);
+
+const datePickerSelector = () => document.querySelector(".usa-date-picker");
+
+const tests = [
+  { name: "document.body", selector: () => document.body },
+  { name: "date picker", selector: datePickerSelector },
+];
+
+tests.forEach(({ name, selector: containerSelector }) => {
+  describe(`date picker in multiple languages ${name}`, () => {
+    const { body } = document;
+
+    let root;
+    let button;
+
+    function changeLanguage(lang) {
+      document.documentElement.lang = lang;
+      window.dispatchEvent(new Event("languagechange"));
+    }
+
+    beforeEach(() => {
+      body.innerHTML = TEMPLATE;
+      DatePicker.on(containerSelector());
+      root = datePickerSelector();
+      button = root.querySelector(".usa-date-picker__button");
+    });
+
+    afterEach(() => {
+      DatePicker.off(containerSelector());
+      window.onerror = null;
+      body.textContent = "";
+      changeLanguage("");
+    });
+
+    it("should display month in english by default", () => {
+      EVENTS.click(button);
+
+      const selectedMonth = root.querySelector(
+        ".usa-date-picker__calendar__month-selection"
+      );
+
+      assert.strictEqual(selectedMonth.innerHTML, "May");
+    });
+
+    it("should display month in the document language", () => {
+      changeLanguage("es");
+
+      EVENTS.click(button);
+
+      const selectedMonth = root.querySelector(
+        ".usa-date-picker__calendar__month-selection"
+      );
+
+      assert.strictEqual(selectedMonth.innerHTML, "mayo");
+    });
+
+    it("should display the correct aria label in the document language", () => {
+      changeLanguage("es");
+
+      EVENTS.click(button);
+
+      const selectedMonth = root.querySelector(
+        ".usa-date-picker__calendar__month-selection"
+      );
+
+      assert.strictEqual(
+        selectedMonth.getAttribute("aria-label"),
+        "mayo. Select month"
```

Comment:
this is the one place where I found additional text that is not able to be translated automatically. I don't think this should be a blocker since it would most likely involve a larger discussion around i18n